pkg/cri/sbserver/podsandbox/sandbox_run.go

```diff
@@ -210,12 +223,13 @@ func (c *Controller) Start(ctx context.Context, id string) (_ *api.ControllerSta
 		return nil, fmt.Errorf("failed to create containerd task: %w", err)
 	}
 	defer func() {
-		if retErr != nil {
+		if retErr != nil && cleanupErr == nil {
```

Comment:
No cleanupErr exists in the following code in this function. Is this just for future-proof?

Reply:
Yes